### PR TITLE
Add pre component

### PR DESCRIPTION
**Pull request by CalebBassham**
Added the pre component so that people can get all of the styles for pre

### CODE REVIEW

File: src/components/pre/pre.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <slot></slot>
+</template>
+
+<script>
+    export default {
+        name: "pre"
+    }
+</script>
+
+<style scoped>
```
Comment (TotomInc) [inline src/components/pre/pre.vue]: I've just merged `style-import` branch, can you please add the style dynamically from the papercss repo, like I have done in other components? Also add `lang="css"` please.

**Review by TotomInc — CHANGES_REQUESTED** [review]

File: src/components/pre/pre.vue (hunk 1 above)
Comment (CalebBassham) [reply src/components/pre/pre.vue]: will do

**Review by TotomInc — APPROVED** [review]

**Comment by TotomInc** [comment]
Can you please point to develop now @DarkPaladinUHC?

**Comment by CalebBassham** [comment]
@TotomInc Sorry, thought I had already done that!

**Pull request merged**